theme/scripts/index.ts

```diff
@@ -1,7 +1,9 @@
-import componentHandler from './componentHandler'
-import InView from './components/InView'
-import Shapes from './components/Shapes'
-import Example from './components/Example'
+import { initializeComponents } from '@mangoweb/scripts-base'
+
 import './plugins'
 
-componentHandler([Example, InView, Shapes], 'initComponents')
+import Example from './components/Example'
+import Shapes from './components/Shapes'
+
+// Sort the components alphabetically…
+initializeComponents([Example, Shapes], 'initComponents')
```

Comment:
Každou komponentu prosím na nový řádek, aby se posléze daly abecedně řadit.

Reply:
Při tomhle počtu bude rozdělení na řádky dělat problém s automatickým formátováním. Jsem pro to spíš takhle nechat.

Reply:
```js
initializeComponents([
	Example, 
	Shapes,
	another,
	great,
	things,
	goHere,
	areWeClear,
], 'initComponents')
```

Reply:
Ano, až jich bude víc, určitě na víc řádků, ale to udělá prettier sám.

Reply:
> printWidth

https://github.com/manGoweb/MangoPress/blob/master/.prettierrc